apps/mobile/src/components/Feed/Posts/PostListCaption.tsx

```diff
@@ -50,14 +53,15 @@ export function PostListCaption({ feedPostRef }: Props) {
   }, []);
 
   const shouldShowGradient = textHeight > 0 && !isExpanded;
+  const gradientColors = colorScheme === 'dark' ? ['rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 1)'] : ['rgba(255, 255, 255, 0)', 'rgba(255, 255, 255, 1)'];
```

Comment:
could we put this in a useMemo? arrays and objects in JS will be recomputed in react unless memoized

better yet could we do something like this:

```ts
// mobile/src/hooks/useGradientColorScheme.ts
export function useGradientColorScheme() {
  const { colorScheme } = useColorScheme();
  return useMemo(() => colorScheme === 'dark' ? ['rgba(0, 0, 0, 0)', 'rgba(0, 0, 0, 1)'] : ['rgba(255, 255, 255, 0)', 'rgba(255, 255, 255, 1)'], [colorScheme])
}

```